grub2_theme_preview/__main__.py

```diff
@@ -308,13 +308,15 @@ def _grub2_ovmf_tuple():
     if the file is missing, as a 2-tuple.
     """
     _DEBIAN = 'Debian GNU/Linux'
+    _Arch_Linux = 'Arch Linux'
     distro_map = {
-        'Arch Linux': ('/usr/share/edk2-ovmf/x64/OVMF_CODE.fd', 'edk2-ovmf'),
+        _Arch_Linux: ('/usr/share/edk2-ovmf/x64/OVMF_CODE.fd', 'edk2-ovmf'),
```

Comment:
Would you mind turning this variable name to all-uppercase `_ARCH_LINUX` or `_ARCH` to match `_DEBIAN` right above? Thanks in advance!

Reply:
Of cource not, that's my negligence!